frontend/packages/integration-tests-cypress/tests/dashboards/project-dashboard.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { checkErrors, testName } from '../../support';
+import { nav } from '../../views/nav';
+
+describe('Project dashboard', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.switcher.changePerspectiveTo('Administrator');
+    nav.sidenav.switcher.shouldHaveText('Administrator');
+    cy.createProject(testName);
+  });
+
+  beforeEach(() => {
+    cy.visit(`/k8s/cluster/projects/${testName}`);
```

Comment:
Just a heads up that I think `cy.visit` starts to navigate to the url, but you might want a `cy.url().should('equal', 'foo')...` (I forget the exact syntax) to provide an additional check before proceeding to each test.

Reply:
Hey, @dtaylor113. Is this a best practice? I'm hesitant to add this after our use of `cy.visit` everywhere unless we know we need it.

Reply:
I 'think' we're ok.

Cypress states:
`cy.visit() resolves when the remote page fires its load event.`
..but is that asychronous?

The test is:
```
cy.visit('/');
nav.sidenav.switcher.changePerspectiveTo('Administrator');
```
The first cmd of `changePerspectiveTo('Administrator')` is a `cy.get` for the switcher legacy id.

cy.get()'s doc is:
```
cy.get() will automatically retry until the element(s) exist in the DOM
cy.get() will automatically retry until all chained assertions have passed
```
So I 'think' we're ok, 'cy.visit' may come back immediately (async) but the `cy.get` will wait until the switcher exists in the DOM, else timeout.  I say 'think' because I also see [this](https://docs.cypress.io/api/commands/get#Arguments) for `cy.get()`:

`When using aliases with DOM elements, Cypress will query the DOM again if the previously aliased DOM element has gone stale.`  -well, how can it go stale if cy.get retries until element exists in the DOM?  Does it retry on a stale DOM?